.github/actions/pkg_installation_testing/action.yml

```diff
@@ -0,0 +1,47 @@
+name: 'Package installation testing'
+description: 'Test that given package can be installed from a given repo'
+inputs:
+  repo_base_url:
+    description: 'base url to the rpm/yum/apt repository of NR'
+  package_name:
+    description: 'name of a package'
+  expected_package_version:
```

Comment:
out of curiosity, why not just `package_version` ? this way it looks a var for assertion